Taskfile.yaml

```diff
@@ -196,6 +196,24 @@ tasks:
         echo "✅ Redeployment complete!"
         echo "Check logs with: task test-infra:kubectl -- logs -n milo-system -l app.kubernetes.io/name=milo-controller-manager"
 
+  dev:install-observability:
+    desc: "Install test-infra observability, then apply Milo-specific scraping manifests"
+    silent: true
+    cmds:
+      - |
+        set -euo pipefail
+        echo "🔧 Installing base observability stack via test-infra…"
+        task test-infra:install-observability
+
+        echo "📎 Applying Milo observability (ServiceMonitor + cert)…"
+        task test-infra:kubectl -- apply -k config/components/prometheus-monitoring/
+
+        echo "⏳ Waiting for vmagent & vmsingle to be Ready (best-effort)…"
+        task test-infra:kubectl -- wait --for=condition=Available deploy -l app.kubernetes.io/name=vmagent -n telemetry-system --timeout=120s || true
+        task test-infra:kubectl -- wait --for=condition=Available deploy -l app.kubernetes.io/name=vmsingle -n telemetry-system --timeout=120s || true
```

Comment:
Is this something that should be done in `task test-infra:install-observability` as well?